src/main/java/com/devathon/preguntonic/model/Room.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * Copyright (c) 2024 Devathon., All Rights Reserved.
+ *
+ */
+package com.devathon.preguntonic.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Room {
+
+  private UUID id;
+  private String code;
+  private RoomStatus status;
+  private Game game;
```

Comment:
this, and https://github.com/Devathon-2024-team5/Preguntonic-backend/blob/602e42421b7d032f32a04cbbd724c9a75807e559/src/main/java/com/devathon/preguntonic/model/Game.java#L22 could cause a cross dependency?

I propose remove it from Game WDYT?

Reply:
Probably will cause a cross dependency, so we can remove parent reference from Game.java